QOLOE-171 Typography QA

tedqgov opened the pull request:
QA issue: https://ssq-qol.atlassian.net/browse/QOLOE-171

duttonw commented on src/scss/qld-type.scss:
```diff
@@ -55,7 +55,7 @@ $link-visited-color:          $qld-link-visited;
     --qld-link-hover-color-rgb: 255, 255, 255;
 
-    --qld-link-visited: var(--qld-link-visited-dark);
+    --qld-link-visited: var(--qld-link-visited-dark)!important;
```
is !important really required?
There should be a better way of doing this.

duttonw commented on src/scss/qld-type.scss:
```diff
@@ -70,6 +70,21 @@ body {
+}
+
+.nav, .nav ul {
+    line-height: 1.5!important;
```
ditto (is there a better way instead of getting out the override hammer)

duttonw commented on src/scss/qld-type.scss:
```diff
@@ -84,6 +99,17 @@ body {
+        a, a.nav-link {
+            text-decoration-color: #B5CCE2;
+            &:visited {
+                color: var(--qld-link-visited-dark)!important;
```
simliar

duttonw reviewed (changes requested):
Can another help out in showing a way to not need !important. It should only be used sparingly and this is clean room, so you should be able to rejig the css to accommodate what you want to do. 

Leave this in will cause much pain in the future.

re-becca commented:
@tedqgov Noticed two minor things for link underline:

- light/alternative - visited:hover should have a text-decoration-color of #551A8B (use associated variable)
- dark/darkalt - default:hover should have a text-decoration-color of #FFFFFF (use associated variable)

tedqgov commented:
@re-becca Alright, I made the changes and cleaned up the code more, also added a new colour variable "$qld-link-dark-underline" in the variables file -- let me know if there's anything else you need :)

tedqgov commented on src/scss/qld-variables.scss:
```diff
@@ -28,8 +28,9 @@ $qld-light-background: #EFF4F9;
 $qld-dark-focus: #01B0E5;
-$qld-link-visited: #8B63B0;
-$qld-link-visited-dark: #E1C2FF;
+$qld-link-visited: #551A8B;
```
@MitchAnderson93 Yeah I noticed when fixing up conflicts, I left the colour in hex code for now in qld-type.scss

tedqgov commented:
Had to reset my commit due to merge conflicts, will take a while

duttonw commented:
Hi @tedqgov ,
Is this included in https://github.com/qld-gov-au/qgds-bootstrap5/pull/214


So this can be closed then?

tedqgov commented:
@duttonw Nope not included, will delete this branch -- I cherrypicked commits to the new branch due to git rebase looping bug.
